### PR TITLE
feat: Created Login and signup

### PR DESCRIPTION
**Pull request by arjunbharti**
Created login and sign up page:

- Login

    1. Email and password
    2. Remember me checkbox
    3. Redirect link to Sign up

- Sign up

    1. Email, password and confirm password
    2. Redirect link to login

Preview Link: https://deploy-preview-2--fakesea.netlify.app/pages/login.html

### CODE REVIEW

File: styles/login-and-signup.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    padding-top: 4rem;
+    width: 100%;
+    height: 100vh;
+}
+
+.login-card{
+    background-color: white;
+    padding: 4rem;
+    height: 500px;
+    width: 400px;
```
Comment (akash-1318) [inline styles/login-and-signup.css:14]: You should not use pixels here.

File: styles/login-and-signup.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    text-align: center;
+    text-decoration: underline;
+    color: var(--dark-v2);
+}
+
+.signup-card{
+    background-color: white;
+    padding: 4rem;
+    height: 550px;
+    width: 400px;
```
Comment (akash-1318) [inline styles/login-and-signup.css:60]: Here also you are using pixel unit.

File: styles/login-and-signup.css (hunk 1 above)
Comment (arjunbharti) [reply styles/login-and-signup.css]: fixed. thanks

File: pages/login.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    <header>
+        <div class="logo-items">
+            <img class="logo" src="../assets/icon.png" alt="icon/img">
+            <a class="header-name" href="#">FakeSea</a>    
+        </div>
+        <input class="nav-search" type="search" placeholder="Search trending NFTs here...">
+        <nav>
+            <div class="nav-action-items">
+                <a class="nav-login-action" href="#login">Login</a>
+                <div class="nav-bookmark-action"><i class="far fa-heart"><span class="badge-icon">0</span></i></div>
```
Comment (darshandamre) [inline pages/login.html:28]: update the login link over here

File: pages/signup.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    <header>
+        <div class="logo-items">
+            <img class="logo" src="../assets/icon.png" alt="icon/img">
+            <a class="header-name" href="#">FakeSea</a>    
+        </div>
+        <input class="nav-search" type="search" placeholder="Search trending NFTs here...">
+        <nav>
+            <div class="nav-action-items">
+                <a class="nav-login-action" href="#login">Login</a>
+                <div class="nav-bookmark-action"><i class="far fa-heart"><span class="badge-icon">0</span></i></div>
```
Comment (darshandamre) [inline pages/signup.html:28]: update the login link

File: pages/login.html [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+                <input 
+                        type="password" 
+                        name=""
+                        id=""
+                        class="input-class"
+                        placeholder="Enter your password"
+                >
+            </div>
+            <div>
+                <input type="checkbox" name="" id="" />
```
Comment (darshandamre) [inline pages/login.html:62]: dont keep these empty attributes or give some value to them, also the `for` attribute in label should be same as its respective input field's `id` value

File: pages/signup.html [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+                        class="input-class"
+                        placeholder="Enter your password"
+                >
+            </div>
+            <div class="login-input-field flex-column">
+                <label for="" class="password-label">Confirm Password</label>
+                <input 
+                        type="password" 
+                        name=""
+                        id=""
```
Comment (darshandamre) [inline pages/signup.html:66]: same here

**Review by darshandamre — COMMENTED** [review]
some suggestions

File: pages/login.html [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+    <!-- page title -->
+    <title>FakeSea | Login</title>
+</head>
+<body>
+    <header>
+        <div class="logo-items">
+            <img class="logo" src="../assets/icon.png" alt="icon/img">
+            <a class="header-name" href="#">FakeSea</a>    
+        </div>
```
Comment (darshandamre) [inline pages/login.html:23]: the link in logo should be `/` so when user clicks on it they are redirected to the home page, same in rest of the pages

File: pages/login.html (hunk 3 above)
Comment (arjunbharti) [reply pages/login.html:28]: updated, thanks

File: pages/signup.html (hunk 4 above)
Comment (arjunbharti) [reply pages/signup.html:28]: updated, thanks

File: pages/login.html (hunk 5 above)
Comment (arjunbharti) [reply pages/login.html]: Updated, thanks

File: pages/login.html (hunk 7 above)
Comment (arjunbharti) [reply pages/login.html]: updated, thanks

**Pull request merged**